Readme.md

```diff
@@ -1,5 +1,7 @@
 # Appfolio base ESLint config
 
+## ** This is config is now deprecated, please use https://github.com/appfolio/front-end-libs ([package](https://github.com/appfolio/front-end-libs/pkgs/npm/eslint-config-appfolio)) **
```

Comment:
It'd be better to link to the subdirectory with the eslint config as it's a monorepo